testsuite/tests/tool-toplevel/pr9701.ml

```diff
@@ -0,0 +1,12 @@
+(* TEST
+ocaml_script_as_argument = "true"
+ocaml_exit_status = "2"
+* setup-ocaml-build-env
+** ocaml
+*)
```

Comment:
You need to add a `check-ocaml-output` step (see for example `tests/typing-shadowing-of-pervasives-submodules/redefine_largefile_top.ml`).

Reply:
Thank you very much

Reply:
Done